core/src/thread_parker/windows/keyed_event.rs

```diff
@@ -73,6 +73,10 @@ impl KeyedEvent {
                 return None;
             }
 
+            // REVIEW: The first hit for this api on google is "Undocumented
+            // functions of NTDLL". The standard library has historically been
+            // conservative to not use undocumented platform features where
+            // possible, such as symbolication APIs for backtraces on OSX.
```

Comment:
This is only used on older versions of Windows (XP, Vista, 7) which don't have `WaitOnAddress`.

Reply:
Ok but this is still missing any documentation like:

* Why is it justified to use an undocumented API?
* What is the general shape of this module and protocol used?
* Are there other systems doing something similar this is mirrored from?  (etc etc)